xctemplates/Router Templates/SwiftUI Screen.xctemplate/___FILEBASENAME___Screen.swift

```diff
@@ -2,16 +2,15 @@
 
 import SwiftUI
 
-protocol ___FILEBASENAMEASIDENTIFIER___Router: AnyObject {
-}
+protocol ___FILEBASENAMEASIDENTIFIER___Router: AnyObject {}
 
 struct ___FILEBASENAMEASIDENTIFIER___: View {
-    @State var router: ___FILEBASENAMEASIDENTIFIER___Router?
-    
     @ObservedObject var viewModel: ___FILEBASENAME___ViewModel
-    
+
+    @State var router: ___FILEBASENAMEASIDENTIFIER___Router?
+
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
```

Comment:
This is a template file, it's ok that it has menu token markers

Reply:
Yes, it's totally ok to have token markers and I can put them back if you'd like.  I just didn't see the value in making someone accept the token default to have a valid generated file, given that they will likely not be needing a Text view at all and will need to put something else there altogether.

Reply:
This is also how the other template that contains a View works.

Reply:
Text Views are the default placeholders on SwiftUI projects. It's fine.

Reply:
I can add this token back in, should I also update the other template to use this as well for consistency sake?

Reply:
yes, that would be good